ios/Podfile.lock

```diff
@@ -672,4 +672,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 41b806c7f131f87b716be1f1f9377532d6c9e43a
 
-COCOAPODS: 1.10.0
```

Comment:
Did you mean to roll back Cocoapods back?

Reply:
I didn't change the lock file. It was an automatic change that was generated by installing pods in ios project.